wristband/__init__.py

```diff
@@ -21,6 +22,62 @@ def wrapper(cls):
 api.route = types.MethodType(api_route, api)
 
 
+def get_all_releases():
+    data = requests.get('{}/apps'.format(releases_endpoint)).json()
+    return data
+
+
+def get_all_releases_of_app_in_env(deploy_env, app_name, releases):
+    releases_for_env = []
+    for release in releases:
+        if deploy_env == release.get("env") and app_name == release['an']:
+            del release['an']
+            del release['env']
+            releases_for_env.append(release)
+    return sorted(releases_for_env, key=lambda k: k['ls'], reverse=True)
+
+
+def get_all_app_names(releases):
+    all_app_names = sorted(set([release["an"] for release in releases]))
```

Comment:
are releases unique? In that case frozeset feels a better data structure to be used